code-challenges.test.js

```diff
@@ -52,6 +89,30 @@ var testArray1 = [3, 7, 10, 5, 4, 3, 3]
 var testArray2 = [7, 8, 2, 3, 1, 5, 4]
 // Expected output: [3, 7, 10, 5, 4, 8, 2, 1]
 
-
+describe("oneArray", () => {
+    test("returns one array with no duplicate values", () => {
+      expect(oneArray(testArray1, testArray2)).toEqual([3, 7, 10, 5, 4, 8, 2, 1])
+    })
+  })
+//Returns oneArray is not defined
 
 // b) Create the function that makes the test pass.
+
+//declare oneArray function
+//concat both input arrays
+// use for loop to iterate through array
+// use if statement to see if current index is strictly equal to previous index
+//push to new array
+//declare new array to accept above values
+//return new array
+const oneArray = () => {
+    let sumArray = testArray1.concat(testArray2)
+    var uniqueArray = []
+    for(i=0; i < sumArray.length; i++){
+        if(uniqueArray.indexOf(sumArray[i]) === -1) {
+            uniqueArray.push(sumArray[i]);
+        }
+    }
+    return uniqueArray
+}
+console.log(oneArray());
```

Comment:
my only only feedback here is that we want to pass the arrays as parameters. Otherwise good work!